corehq/apps/cleanup/tasks.py

```diff
@@ -1,10 +1,47 @@
+import os
+import json
+from time import time
 from collections import defaultdict
 
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.core.management import call_command
+
+from corehq.apps.hqwebapp.tasks import mail_admins_async
 from corehq.apps.cleanup.management.commands.fix_xforms_with_undefined_xmlns import \
     parse_log_message, ERROR_SAVING, SET_XMLNS, MULTI_MATCH, \
     CANT_MATCH, FORM_HAS_UNDEFINED_XMLNS
 
 
+UNDEFINED_XMLNS_LOG_DIR = '/tmp/'
```

Comment:
that's fair, i'll move them to place a little more permanent. fwiw `/tmp` only gets cleared on reboot, which does not happen very often on our machines (or at least shouldn't). though storing it in `/tmp` is like saying it's fine to delete whenever, which is probably more of a concern